aggregation/src/idl/aggregations.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto2";
+option java_package = "fk.prof.aggregation.proto";
+option java_outer_classname = "AggregatedProfileModel";
+
+message ProfileSourceInfo {
+	optional string ip = 1;
+	optional string hostname = 2;
+	optional string processName = 3;
+	optional string zone = 4;
+	optional string instanceType = 5;
+}
+
+enum ProfileStatus {
+	Completed = 0;	// got all the samples
+	Partial = 1;	// got only few of the samples.
+	Retried = 2;	// may have duplicated samples because of retries 
+	Aborted = 3;    // {see if its required}.
+}
+
+message ProfileSummary {
+	optional string startTime = 1;				// HH:mm:ss
```

Comment:
I really think uint32 would have been better. Because it implicits start time (its basically start offset from aggregation-window start and end offset from aggregation window start), or it can be length from capture-window start.

iso8601 allows for a larger family of bugs, which uint32 doesnt.